docs/development.md

```diff
@@ -63,12 +63,12 @@ accomplished as follows:
 
 ```bash
 # Bootstrap and commit a new type
-$ kubebuilder create api --group <your-group> --version v1alpha1 --kind <your-kind>
+$ kubebuilder create api --group <your-group> --version v1beta1 --kind <your-kind>
```

Comment:
I don't think this should change. Bootstrapping a new API group should still land into alpha.